tests/dummy/app/router.js

```diff
@@ -24,6 +24,11 @@ Router.map(function() {
 
   this.alias('alias-one', '/alias-one', 'one');
   this.alias('not-one', '/not-one', 'alias-one');
+
+  this.route('non-terminal', function() {
+    this.route('index', { path: '/', terminal: 'non-terminal.terminal' });
```

Comment:
Without path, should the expected behavior be to redirect "/non-terminal/index" or just assume "/" and redirect "/non-terminal"?